src/SFA.DAS.AssessorService.Data/CertificateRepository.cs

```diff
@@ -193,7 +204,8 @@ private bool CheckCertificateData(Certificate certificate, string lastName, Date
 
         public async Task<List<Certificate>> GetCompletedCertificatesFor(long uln)
         {
-            return await _context.Certificates.Where(c => c.Uln == uln && (c.Status == CertificateStatus.Reprint || c.Status == CertificateStatus.Printed || c.Status == CertificateStatus.Submitted || c.Status == CertificateStatus.ToBeApproved))
+            var statuses = new[] { CertificateStatus.Submitted, CertificateStatus.ToBeApproved }.Concat(CertificateStatus.PrintProcessStatus).ToList();
```

Comment:
are these collectively known as "Completed statuses"?

Reply:
Recently I have been trying to collect together status sets in the certificate status class - it is possible that this set of status represents a completed status, it is not used else where in this combination - I'm not sure if that does make it a collective status or not? I'm thinking that where it is used at least twice it could be promoted

Reply:
I was just thinking that the variable could be named as such for now to make it apparent. I made the connection based on the method name but would maybe be better to be explicit

Reply:
Ok - will rename the variable for clarity